.github/workflows/sync-repo-labels.yml

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
what's this blank line

Reply:
Was just in my example file, have removed.